build/azDevOps/azure/azure-pipelines-netcore-k8s.yml

```diff
@@ -698,4 +698,5 @@ stages:
           # Push the package to Nuget
           - template: azDevOps/azure/templates/v2/steps/deploy-push-package-symbols-public.yml@templates
             parameters:
-              nuget_sevice_connection: $(nuget_service_connection)
+              nuget_sevice_connection: NuGetAmidoStacksServiceConnection
```

Comment:
Please do not use a static string here, we need to establish where the var `$(nuget_service_connection)` is being declared and update the value inside that. @alexanderwjrussell has determined this isn't in any of the files in the repo, but may be in a linked variable group (syntax: `- group: xxx`). Other repos declare this at the top level of the pipeline file in the main variable list.

Reply:
Some tasks require the connection upfront when the task is initialised just after template compile which means `$(...)` can't be used, if that's the error you're getting it should be possible to use `${{ ... }}` instead which happens at template compile time.

I'm not sure if that's the case here.

Reply:
@dibyadhar Fyi that is `${{ variables.xxx }}`  - @ElvenSpellmaker the syntax hasn't changed, it was originally the `$(...)` format, just that it appears the value has gone missing!

Reply:
@williamayerst It's just that some tasks require the value upfront and the Pipeline will crash saying it can't find the service connection, I was just wondering if that was the problem, nevermind if not!

Reply:
I am 100% with you on that one, but did not appear to be the case as it was/appeared to be functioning using the runtime syntax.